src/client/containers/SignIn/index.js

```diff
@@ -11,5 +14,21 @@ export default function SignInContainer() {
     setIsLoading(false);
   };
   if (isLoading) return <Loader />;
-  return <SignIn onSubmit={onSubmit} />;
+  return (
+    <div>
+      <div className="split sign-in-left">
+        <div>
+          <SideBanner
+            image={signInImg}
+            style={{ backgroundColor: '#6236FF' }}
```

Comment:
You can not use a style attribute on a component.  You can use it on a <div> or some other element but not on component <SideBanner/> like you did.

Reply:
can you look at SideBanner component @ember-b-Moss and @pdsorensen 
SideBanner.propTypes = {
  image: PropTypes.string.isRequired,
  style: PropTypes.shape({ backgroundColor: PropTypes.string }).isRequired,
};

Reply:
I haven´t seen you made PropTypes in any of the files changed @kamalrsa ? So, I´m not sure what are you asking here?

Reply:
My bad 😶 The issue is actually because I created the component SideBanner, which takes the props that Kamal showed above. So, I will make a PR to remove the style prop on that component. 
We actually use JS styling in the Button as well, which is from where I took inspiration for passing the style as props but now I see that it is not recommended in [here](https://reactjs.org/docs/dom-elements.html#style). sorry!